universum/lib/gravity.py

```diff
@@ -93,7 +93,10 @@ class Dependency(Generic[DependencyType]):
     def __init__(self, cls: Type[DependencyType]) -> None:
         self.cls = cls
 
-    def __get__(self, instance: DependencyType, owner: Any) -> Callable[..., DependencyType]:
+    # the source and the target of the dependency are different modules,
+    # so wee need to use different type variables to annotate them
+    # instance parameter of the __get__ method is source module
+    def __get__(self, instance: ComponentType, owner: Any) -> Callable[..., DependencyType]:
```

Comment:
does it add any value that we use 'ComponentType' here instead of just 'Module'?

Reply:
I've checked it with the Module and it looks like it is also ok.